src/Evolution/Systems/GeneralizedHarmonic/Characteristics.hpp

```diff
@@ -59,6 +59,20 @@ namespace GeneralizedHarmonic {
  * surface.
  */
 template <size_t Dim, typename Frame>
+typename Tags::CharacteristicSpeeds<Dim, Frame>::type characteristic_speeds(
```

Comment:
The changes in `src/Evolution/Systems/GeneralizedHarmonic` should be in a seperate commit.